### ISSUE TITLE
Circular Camera set up

### ISSUE THREAD

**Issue by santiago2389** [oliver-batchelor/multical#70]
The camera set up shown in ReadMe is linear.
Can we calibrate the circular set up accurately using this library?
![IMG20241015142843](https://github.com/user-attachments/assets/e2e06c3c-9c4b-40b3-99ee-833b77b6e9f1)

**Comment by oliver-batchelor** [comment]
Yes that should be fine, not all the cameras need to overlap with every
other camera.
